src/openloco/ui/dropdown.cpp

```diff
@@ -60,9 +70,24 @@ namespace openloco::ui::dropdown
         add(index, title, { l });
     }
 
+    int16_t get_highlighted_item()
+    {
+        return _dropdownHighlightedIndex;
+    }
+
+    void set_disabled_item(int16_t index)
+    {
+        _dropdownDisabledItems = _dropdownDisabledItems | (1 << index);
```

Comment:
`set_item_disabled` would make more sense to me
`set_item_disabled` feels like `dropdown->items[i].disabled`, whereas `set_disabled_item` feels like `dropdown->disabledItem = i`.